src/js/components/JobForm.js

```diff
@@ -58,6 +56,33 @@ class JobForm extends SchemaForm {
     this.props.onTabChange(tab);
   }
 
+  validateForm() {
+    let model = this.triggerTabFormSubmit();
+
+    let validated = true;
+    // Apply all validations.
+    FormUtil.forEachDefinition(this.multipleDefinition, function (definition) {
+      definition.showError = false;
+
+      if (typeof definition.externalValidator !== 'function') {
+        return;
+      }
+
+      let fieldValidated = definition.externalValidator(model, definition);
+      if (!fieldValidated) {
+        validated = false;
+      }
+    });
+
+    this.forceUpdate();
+
+    return {
```

Comment:
I know that this mirrors the `ServiceForm`, but do we actually need the return? Is there no way to simply validate the form and the return the data using `getDataTriple`?